src/components/charts/Histogram.js

```diff
@@ -0,0 +1,29 @@
+import React, { useMemo } from "react";
+import PropTypes from "prop-types";
+import { BarChart } from "bento-charts";
+import ChartContainer from "./ChartContainer";
+
+const transformData = (data) => data && data.map(({ ageBin, count }) => ({ x: ageBin, y: count }));
+
+const Histogram = ({ title = "Histogram", data = [], chartHeight = 300 }) => {
+    const transformedData = useMemo(() => transformData(data), [data]);
+
+    return (
+        <ChartContainer title={title} empty={!Array.isArray(data) || !data.length}>
+            <BarChart data={transformedData} height={chartHeight} units="" />
+        </ChartContainer>
+    );
+};
+
+Histogram.propTypes = {
+    title: PropTypes.string,
+    data: PropTypes.arrayOf(
+        PropTypes.shape({
+            ageBin: PropTypes.string.isRequired,
```

Comment:
We may want to display histogram data that is not related to patients age, so I would make the props more generic.
Since `transformData` only maps `ageBin` to `x` and `count` to `y` I think we can simply use those as our prop types and get rid of `transformData`.